### ISSUE TITLE
Fonction save verset

### ISSUE THREAD

**Issue by fedwiiix** [HackMyChurch/aelf-dailyreadings#32]
Vous pourriez ajouter une fonction pour sauvegarder des versets qui nous touchent... Une amie utilise une application protestante pour cette fonction...

**Comment by nath5394** [comment]
Yes @yadutaf est en train de réécrire la partie Bible en natif. On pourra faire ça après plus facilement. Comme l'application est open source tu peux regarder. Si ça te dis de travailler sur cette fonctionnalité !!! :)

**Comment by iamSangohan** [comment]
La fonctionnalité est-elle toujours en cours ?

**Comment by nathanael-h** [comment]
Je crois pas. 
Sur cette app native on passe en maintenance. Et on essaie de faire avancer la version flutter dispo sur iOS qui pourrait être multiplateforme. 
Elle est ici https://gitlab.com/nathanael2/aelf-flutter

**Comment by iamSangohan** [comment]
Ah ok. C'est cool, je fais un peu de Flutter, je pourrais vous aidez sur cette fonctionnalité dans mon temps libre

**Comment by nathanael-h** [comment]
Oh yeah, coool contact moi en privé si tu veux je te présenterai le contexte !! :rocket:

**Comment by iamSangohan** [comment]
😊 c'est compris
